lxd/db/instance_profiles.go

```diff
@@ -30,3 +34,43 @@ type InstanceProfileFilter struct {
 	InstanceID *int
 	ProfileID  *int
 }
+
+// UpdateInstanceProfiles updates the profiles of an instance in the order they are given.
+func (c *ClusterTx) UpdateInstanceProfiles(instance Instance) error {
+	err := c.DeleteInstanceProfiles(instance)
+	if err != nil {
+		return err
+	}
+
+	project := instance.Project
+	enabled, err := projectHasProfiles(c.tx, project)
+	if err != nil {
+		return errors.Wrap(err, "Check if project has profiles")
```

Comment:
Is it one of those cases where we can't use fm.Errorf with %w? If not, we should really stay away from pkg/errors.

Reply:
@stgraber, this is just a remnant of copy/pasting. I'll switch it.